server/libs/infra/src/infra.config.ts

```diff
@@ -26,17 +26,24 @@ function parseRedisConfig(): RedisOptions {
 
 export const redisConfig: RedisOptions = parseRedisConfig();
 
-export const bullConfig: BullModuleOptions = {
+interface CustomBullModuleOptions extends BullModuleOptions {
+  onError?: (error: any) => void;
+}
+
+export const bullConfig: CustomBullModuleOptions = {
   prefix: 'immich_bull',
   redis: redisConfig,
   defaultJobOptions: {
     attempts: 3,
     removeOnComplete: true,
     removeOnFail: false,
   },
+  onError: (error) => {
+    console.log(error);
```

Comment:
Can this use the nestjs logger instead of console.log?

Reply:
Should be able to instantiate a new logger in this life and then use it here.

Reply:
That's a good question as it's not inside a class so one cannot simply do this.logger, but should be possible.